### DISCUSSION TITLE
Optional need urgent help on my Ui accessibility

### DISCUSSION THREAD

**Discussion by Daniel477-creator** [community/community#43780] [Accessibility]
Help

**Comment by radar-blip** [comment]
What do you need help with?

**Comment by ghostinhershell** [comment]
Hi @Daniel477-creator, welcome to the GitHub Community! We're happy you're here 😄

You are more likely to get a useful response if you are explicit about what your project entails and what you need help with. Giving a few more details might help someone give you a nudge in the right direction.
